core/modules/utils/dom/dom.js

```diff
@@ -268,9 +268,7 @@ exports.copyStyles = function(srcDomNode,dstDomNode) {
 /*
 Copy plain text to the clipboard on browsers that support it
 */
-exports.copyToClipboard = function(text,options) {
-	options = options || {};
-	text = text || "";
+exports.copyToClipboard = function(text = "",options = {},type = "text/plain") {
```

Comment:
This line is not js-code backwards compatible. Initialisation within function calls is ES6+

ATM we use the mechanism as `options = options || {};` if we need to initialize functions call parameters.